### PR TITLE
[OS-1036] Fix URL for AppStore in Dashboard icon and kano-apps

### PR DESCRIPTION
**Pull request by skarbat**
Fix the old URL by adding a simple wrapper script which:

 * Opens the browser pointing to the new url coming from the kano-world.conf file
 * Adds an hourglass until the browser is interactive
 * Removes the pre-login authentication which is not needed anymore
 * Changes the Dashboard and Kano-apps launch commands to call this script

I have tested the Python side on the RPi - working fine.

### CODE REVIEW

File: bin/kano-apps-store-web [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+if __name__ == '__main__':
+
+    browser = 'chromium-browser'
+    kano_store_url = load_conf()['store_url']
+    command_line = '{} ' \
+                   '--start-maximized ' \
+                   '--force-device-scale-factor=1.2 {}'.format(browser, kano_store_url)
+
+    os.system('kdesk-hourglass-app {}'.format(browser))
+    rc = os.WEXITSTATUS(os.system (command_line))
```
Comment (radujipa) [inline bin/kano-apps-store-web]:
```suggestion
    rc = os.WEXITSTATUS(os.system(command_line))
```

File: bin/kano-apps-store-web [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+#
+# kano-apps-store-web
+#
+# Copyright (C) 2014-2018 Kano Computing Ltd.
```
Comment (radujipa) [inline bin/kano-apps-store-web]:
```suggestion
# Copyright (C) 2019 Kano Computing Ltd.
```

**Comment by radujipa** [comment]
@skarbat could you also bump the changelog version to `4.3.1` as well please?

File: bin/kano-apps-store-web (hunk 2 above)
Comment (skarbat) [reply bin/kano-apps-store-web]: Done

File: bin/kano-apps-store-web (hunk 1 above)
Comment (skarbat) [reply bin/kano-apps-store-web]: Fixed

File: debian/changelog [hunk 3]
```diff
@@ -1,8 +1,9 @@
-kano-apps (4.2.1-0) unstable; urgency=low
+kano-apps (4.3.1-0) unstable; urgency=low
```
Comment (radujipa) [inline debian/changelog]: Let's add a new entry altogether rather than editing the old one

Comment (skarbat) [reply debian/changelog]: Done

**Pull request merged**